src/frontend/src/components/Header.css

```diff
@@ -16,3 +16,12 @@ body {
   float: left;
   padding-right: 5px;
 }
+
+.header-search-form {
```

Comment:
I don't think this would target the class you are trying to target.

Reply:
Fixed!

Reply:
FTFY